ui/utils/camelCaseToSentanceCase.js

```diff
@@ -1,3 +1,3 @@
-export const CamelCaseToSentanceCase = (stringValue) => {
+export const CamelCaseToSentenceCase = (stringValue) => {
```

Comment:
Suggestion: anything which is not a component or enum, usually should not be starting with an upper case.